sw_utils/ipfs.py

```diff
@@ -238,10 +240,20 @@ async def remove(self, ipfs_hash: str) -> None:
 
 
 class IpfsFetchClient:
-    def __init__(self, endpoints: list[str], timeout: int = 60):
+    def __init__(
+            self,
+            endpoints: list[str],
+            timeout: int = 60,
```

Comment:
```suggestion
            retry_timeout: int = 60,
```

Reply:
`backoff_in_method` was deleted

Reply:
I have deleted `request_timeout` param. Let's deal with ipfs in another PR.